lib/reflifc/include/reflifc/Module.h

```diff
@@ -54,6 +65,24 @@ namespace reflifc
                 });
         }
 
+        ViewOf<Module> auto exported_modules(ifc::Environment& environment) const
```

Comment:
Same here. Let's introduce method
`ViewOf<Module> auto dereference(Partition<ModuleReference>, ifc::Environment&) const`
and reuse it in both `exported/imported_modules` methods.